### ISSUE TITLE
Panic in get_alternatives

### ISSUE THREAD

**Issue by luteberget** [peterheesterman/chit#35]
Hi, I just came across this tool and it looks cool! The README suggests installing with `cargo install chit`, but then it panics on loading the alternatives.json file.

**Describe the bug**
Panic in get_alternatives.

**To Reproduce**
1. `chit which`
2. panic

**Expected behavior**
No panic message.

**Screenshots**
```
$ chit which
Searching for which...
─which──────────────────────────────────────────────
A Rust equivalent of Unix command \"which\". Locate 
installed execuable in cross platforms.

Rating: 🌟 🌟 🌟 🌟 🌟 
Total downloads: 499439
Recent downloads: 254457
Latest version: 2.0.1 (2018-11-27)
Docs: https://docs.rs/which/
Repository: https://github.com/harryfei/which-rs.git
Crates.io: https://crates.io/crates/which
License: MIT
Crate size: 7 kB
thread 'main' panicked at 'file not found: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/libcore/result.rs:999:5
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

```

**Desktop (please complete the following information):**
 - OS: Ubuntu Linux 
 - Version 0.1.9 installed using `cargo install chit` using cargo 1.36.0-nightly (b6581d383 2019-04-16)

**Comment by 8176135** [comment]
Seems like it always expects a `alternatives.json` file to be in the directory you run chit from, which is a bit unreasonable. 

Seems like the short term fix would be to include_string!() the alternatives file into chit.

Long term probably it should be getting the alternative.json from a remote host, like straight from https://raw.githubusercontent.com/peterheesterman/chit/master/alternatives.json, for the latest version, since it is connecting to the internet anyway.

I can probably make a PR for this

**Comment by peterheesterman** [comment]
@luteberget Thanks for trying the tool and finding this! Oversight on my part for sure.

@8176135 Thanks so much for this PR! Will give it some 👀 soon and release a new version accordingly.

**Comment by peterheesterman** [comment]
I have merged the PR from @8176135! Also, i have released v0.1.10 and verified that this is no longer an issue @luteberget, can i get you to confirm that is the case for you? `cargo install chit --force`

**Comment by luteberget** [comment]
It works. Thanks!
